### ISSUE TITLE
Conflict with markdown-toc > Uncaught TypeError: _this.resultNode.scrollIntoView is not a function

### ISSUE THREAD

**Issue by LeoUfimtsev** [mark-hahn/markdown-scroll-sync#32]
I found that markdown-scroll-sync throws an uncaught error if the first thing in the file is a toc that is generated by markdown-toc like:

    

    - [workaround](#workaround)
    - [Socket programming](#socket-programming)
    	- [socketeering](#socketeering)
    	- [linux](#linux)
    

I found that I could work around that if I put my TOC below a heading

# my toc  
        <!-- TOC 

[Enter steps to reproduce below:]
1. Install markdown-toc (0.3.0 at time of writing).
2. Insert toc (search for command, insert toc)
3. Fill multiple pages so that one has to scroll 
4. Scroll down on markdown file.
5. Scroll up
6. When reaching the toc section, this error occurs. 

**Atom Version**: 1.0.16
**System**: Mac OS X 10.10.4
**Thrown From**: [markdown-scroll-sync](https://github.com/mark-hahn/markdown-scroll-sync) package, v0.4.2

### Stack Trace

Uncaught TypeError: _this.resultNode.scrollIntoView is not a function

```
At /Users/lufimtse/.atom/packages/markdown-scroll-sync/lib/main.coffee:116

TypeError: _this.resultNode.scrollIntoView is not a function
  at /Users/lufimtse/.atom/packages/markdown-scroll-sync/lib/main.coffee:116:21
  at MarkdownScrlSync.module.exports [as roaster] (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/roaster/lib/roaster.js:69:14)
  at MarkdownScrlSync.scroll (/Users/lufimtse/.atom/packages/markdown-scroll-sync/lib/main.coffee:104:6)
  at /Users/lufimtse/.atom/packages/markdown-scroll-sync/lib/main.coffee:81:10

```

### Commands

```
 10x -1:36.1.0 markdown-writer:insert-new-line (atom-text-editor.editor.is-focused)
  4x -1:21.3.0 core:backspace (atom-text-editor.editor.is-focused)
  2x -1:18.7.0 markdown-writer:insert-new-line (atom-text-editor.editor.is-focused)
  2x -1:16.8.0 core:backspace (atom-text-editor.editor.is-focused)
     -1:10.3.0 find-and-replace:select-all (atom-text-editor.editor.is-focused)
     -0:53.7.0 fuzzy-grep:toggle (atom-text-editor.editor.is-focused)
     -0:49 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -0:49 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:48.6.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:45.7.0 fuzzy-grep:toggle (atom-text-editor.editor.is-focused)
 10x -0:40.7.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:15.7.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:13.4.0 markdown-preview:toggle (atom-text-editor.editor.is-focused)
 17x -0:07.8.0 markdown-writer:insert-new-line (atom-text-editor.editor.is-focused)
     -0:04.8.0 autocomplete-plus:confirm (atom-text-editor.editor.is-focused.autocomplete-active)
  3x -0:04.5.0 markdown-writer:insert-new-line (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "markdown-folder"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-fuzzy-grep, v0.6.0
markdown-mindmap, v0.2.4
markdown-scroll-sync, v0.4.2
markdown-toc, v0.3.0
markdown-writer, v2.0.0
minimap, v4.13.4
minimap-codeglance, v0.4.2

# Dev
No dev packages
```

**Comment by anothermh** [comment]
+1

**Comment by mark-hahn** [comment]
I've released a new version 2.0.1 that is a total rewrite.  I think it fixes all issues and works much better than before.  Please try it.
